imager/imagerprofile/apps.py

```diff
@@ -0,0 +1,10 @@
+from django.apps import AppConfig
+
+
+class ImagerProfileConfig(AppConfig):
+
+    name = 'imagerprofile'
+    verbose_name = 'Imager Profile'
+
+    def ready(self):
+        from handlers import add_profile
```

Comment:
since you don't actually use 'add_profile' here, it would be better simply to import the module:

```python
def ready(self):
    import handlers
```